site/disk-alarms.xml

```diff
@@ -83,12 +83,18 @@ Disabling disk free space monitoring</pre>
         required to be free on the database partition. (See the description of
         <a href="/relocate.html">file locations</a> for the default location).
```

Comment:
Default database location.

> On 22 mar 2016, at 21:48, Brandon Shroyer <notifications@github.com> wrote:
> 
> In site/disk-alarms.xml:
> 
> > @@ -83,12 +83,18 @@ Disabling disk free space monitoring</pre>
> >          required to be free on the database partition. (See the description of
> >          <a href="/relocate.html">file locations</a> for the default location).
> I'm not sure from the context here what the "default location" is referring to. Is it the free disk space limit?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>